scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/dao/ScheduleDaoImpl.java

```diff
@@ -17,11 +18,17 @@ public ScheduleDaoImpl() {
 		super(ScheduleEntity.class);
 	}
 
+	@SuppressWarnings("unchecked")
```

Comment:
You are using the wrong form of createNamedQuery. Use the other version and the unchecked warning disappears.